server/app/db/crud/bot_types.py

```diff
@@ -0,0 +1,206 @@
+"""Database layer for 'bot_types' Table"""
+import logging
+from typing import List, Tuple, Union
+
+from aiomysql.cursors import Cursor
+from db.errors import RecordNotFound
+from pydantic import ValidationError
+from pypika import MySQLQuery, Table
+
+import server.app.db.schemas.bot_types as bot_type_schema
+
+from .base import log_query
+
+bot_types = Table("bot_types")
+
+
+async def insert_bot_type(
```

Comment:
adding it here for later reference, so we should register bot user in dalal server while inserting in db right ?